core/model/modx/modprocessor.class.php

```diff
@@ -137,7 +137,7 @@ public function addFieldError($key,$message = '') {
      * @param modX $modx A reference to the modX object.
      * @param string $className The name of the class that is being requested.
      * @param array $properties An array of properties being run with the processor
-     * @return The class specified by $className
+     * @return string The class specified by $className
```

Comment:
Sorry, you are right: the include returns the class name. This method returns the class.